docs/docusaurus/docs/cloud/about_gx.md

```diff
@@ -52,6 +52,10 @@ The GX Agent is an intermediary between GX Cloud and your organization's data st
 
 The GX Agent is typically deployed in your organization's deployment environment, for example, in a development, staging, or production cloud services environment. The GX Agent serves all GX Cloud users within your organization. It can be run as part of your development or production workflows. See [Connect GX Cloud](./connect/connect_lp.md) for setup instructions.
 
+### GX Agent versioning
+
+GX uses a date-based versioning format for its weekly GX Agent releases. For example, `YYYYMMMDD`. GX appends `stable` or `dev` to the GX Agent version to identify the release type. The `stable` identifier indicates the release is fully tested and ready for use. The `dev` identifier indicates the release is pre-production and untested. This documentation assumes you are using the latest `stable` GX Agent release. 
```

Comment:
How about something like this?
```suggestion
GX uses a date-based versioning format for its weekly GX Agent releases. For example, `YYYYMMMDD.#` for stable releases or `YYYYMMDD.#.dev#` for pre-releases. GX also provides the `stable` and `dev` docker tags that reference the most recent of each release type. The `stable` identifier indicates the release is fully tested and ready for use. The `dev` identifier indicates the release is a pre-release. This documentation assumes you are using the latest `stable` GX Agent release. 
```

Reply:
GX uses a date-based versioning format for its weekly GX Agent releases. For example, `YYYYMMMDD.#` for stable releases or `YYYYMMDD.#.dev#` for pre-releases. GX uses the `stable` and `dev` identifiers to identify the release type. The `stable` identifier indicates the release is fully tested and ready for use. The `dev` identifier indicates a pre-release. This documentation assumes you're using the latest `stable` GX Agent release.

Reply:
@anthonyburdi @kwcanuck thank you for the suggestions here. I have a preference for the second one because it explains the image tags more specifically.

Reply:
Sounds good to me. @Shinnnyshinshin I'm seeing two different versions of the text in this file though, can you update?